.gitignore

```diff
@@ -348,3 +348,9 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+
+#github folder
+.github
+
+#ignore test.md
+test.md
```

Comment:
- [ ] it would be nice to equally add **_node_modules** directory here. This prevents us from committing this directory to our repo by error since it contains lots of packages.

Reply:
As I've created the repo with a .gitignore file already (using Visual Studio template), the node_modules/ are already in the .gitignore file.

So I've just deleted .github from there.